documentation/__README.md

```diff
@@ -0,0 +1,25 @@
+## Guidelines for maintaining documentation.
+
+The `./documentation` folder is the root documentation folder for this repository.
+
+Within this documentation folder is a clone of the repo's [**wiki**](https://github.com/openmsupply/application-manager-web-app/wiki) repository, which appears as a submodule in the main repo, in `./documentation/_wiki`
+
+The wiki contents is essentially exactly the same as the main documentation folder. However, the way the wiki software handles internal links when online is different, and all pages are presented at the wiki root level, which means some relative links need to be adjusted accordingly.
+
+In order to update the wiki from the current documentation state, and push the wiki docs online, simply run:
+
+`yarn push_docs`
+
+(This runs a python script to convert the wiki docs -- it can be configured by adjusting the variables at the top of `./utils/convert_docs_to_wiki.py`)
```

Comment:
Also it would be nice to add the how to use the submodule git commands:

## Usage
* Clone repo with submodule pulled (if already in master):
`git clone --recurse-submodules https://github.com/openmsupply/application-manager-web-app.git`

* Clone repo with submodule in specific branch:
`git clone --recurse-submodules -b <branch_name> https://github.com/openmsupply/application-manager-web-app.git`

* Init and pull submodule folder:
`git submodule update --init`

1) If already on master, you can clone + pull submodules:
git clone --recurse-submodules https://github.com/openmsupply/application-manager-server.git

2) Or after changing branches the command to init and pull:
git submodule update --init

Reply:
Done.